auslib/blobs/schemas/systemaddons.yml

```diff
@@ -18,9 +18,6 @@
       description: Blob identifier number
       enum:
         - 5000
-    hashFunction:
-      type: string
-      description: Hash function of files pointed at (returned in responses generated by this Blob)
```

Comment:
Sorry - I think I was unclear about this. I meant to remove hashFunction from the "required" section, not from the schema altogether. Removing it here will break any System Addon blobs that actually do have addons, because hashFunction will not be allowed in the blob.